druid/src/widget/images.rs

```diff
@@ -0,0 +1,259 @@
+// Copyright 2019 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! An Image widget.
+//! Please consider using Image and the Image wideget as it scales much better.
```

Comment:
```suggestion
//! Please consider using Image and the Image widget as it scales much better.
```

Also I'm not totally sure what this comment is trying to say? better than what?